packages/@tinacms/cli/src/buildTina/attachDatabase.ts

```diff
@@ -105,8 +126,12 @@ export const attachDatabase = async (
         )
       : fsBridge
 
-    const level = new MemoryLevel<string, Record<string, any>>({
-      valueEncoding: 'json',
+    const level = new ManyLevelGuest()
```

Comment:
I think we can use the TinaLevelClient here now

Reply:
Updated!